src/cart/crt_init.c

```diff
@@ -96,6 +96,8 @@ dump_opt(crt_init_options_t *opt)
 		D_INFO("thread mode single is set\n");
 	if (opt->cio_progress_busy)
 		D_INFO("progress busy mode is set\n");
+	if (opt->cio_mem_device)
+		D_INFO("memory device mode is set\n");
```

Comment:
Those should be probably printed on line 88. we print auth key separately due to security concerns

Reply:
I was trying to understand what you were asking but I still don't get it, can you please look at it again ? I don't think that makes any sense...

Reply:
see above, we print all other boolean flags as: fi: %d, use_credits: %d, use_esnsors: %d\n", opt->cio_fault_inject,
	       opt->cio_use_credits, opt->cio_use_sensors);
	      
'auth_key' is really the only one that is supposed to be handled differently, but for some reason we started printing all flags after that similar to auth-key